src/libstd/fs.rs

```diff
@@ -422,6 +422,11 @@ impl OpenOptions {
     /// This option, when true, means that writes will append to a file instead
     /// of overwriting previous contents.
     ///
+    /// Note that this isn't the same as `write` with the cursor starting at
+    /// the end of the file; this mode only allows writing to the end of the
```

Comment:
I guess I'm not sure of the right way to word it, you're suggesting dropping the 'cursor' bit, I guess? Because appending to a file is still writing to it...